python/GafferSceneUI/SceneInspector.py

```diff
@@ -1675,6 +1768,12 @@ def supportsInheritance( self ) :
 
 			return True
 
+		def supportsShaderParameters( self, target ) :
```

Comment:
This isn't really working as an abstraction, because the `__showShaderParameters` is still assuming that it's an attribute inspector, and then getting the attribute name from `self.__label().getText()`. Since it isn't really an abstraction, I think we'd be better just moving the `isinstance( value, IECore.ObjectVector )` test directly into the menu building code.

I'll make a related comment on `_ShaderParameterSection.__Inspector`...

Reply:
Addressed in cb50163. Thanks!